Suicide Messages, Combat Messages, Uniforms Recovered

Ragolution opened the pull request:
Objectives:
- Fix stripper outfits from the kinkmate and a handful of other uniforms we don't use.
- Add TLC to dildos

Implements Features:
- Adds suicide messages to dildos. Ever wanted to show everyone how good you choke on dick? Granted.
- Adds attack verbs for dildos, with such variations as 'penetrated' and 'knotted' respective to subtype.
- Added flag to old 5th Port/VG uniforms that were broken. Kind of mad about how easy it was and that it took literally a year to fix.
- Maybe doesn't break anything.
- Incorporates and iterates upon Spellchecks done by PR #7892 

Speedmerge candidate IMO

Ragolution committed:
Suicide_Acts for Dildos

By request.

Ragolution committed:
Fixes all vg/5th port uniforms that aren't commented out for whatever reason.

I'm pissy that this was so easy and nobody else bothered to do it.

Ragolution committed:
Improved attack verbs on dildos+ suicide improvements

Adds a bracket that kept the suicide message from appearing and grants each dildo its own attack message, mostly.

Ragolution committed:
Spellcheck 2 Electric Boogaloo

statis

Misspelling of status.

Ragolution committed:
TRAVIS LEAVE ME ALONE

Ragolution committed:
Adjusts wording for grammar on cryo shotgun dart

Travis is fucking bullying me.

Ragolution committed:
I guess it didn't like that.

I don't know how to fix this so I'm changing it back uwu

deathride58 commented on modular_citadel/code/modules/arousal/toys/dildos.dm:
```diff
@@ -131,3 +134,58 @@ obj/item/dildo/custom
+	user.Stun(150)
+	user.adjust_blurriness(8)
+	user.adjust_eye_damage(10)
+	sleep(15)
```
`Sleep()` is obsolete and can degrade server performance during stressful scenarios. Use callbacks instead

Ragolution committed:
Conforms with changes requested by the BhjinBot

Really cool bot I can't believe we have something so advanced for our codebase.

Ragolution committed:
Code improvements

Thanks to @Ghommie for the suggestion.

Ghommie commented on modular_citadel/code/modules/arousal/toys/dildos.dm:
```diff
@@ -142,25 +142,7 @@ obj/item/dildo/custom
-
-/obj/item/dildo/plain/suicide_act(mob/living/user, delay = 150)
+/obj/item/dildo/suicide_act(mob/living/user)
 	user.visible_message("<span class='suicide'>[user] tears-up and gags as they shove [src] down their throat! Their face is turning blue! It looks like [user.p_theyre()] trying to commit suicide!</span>")
```
Simply add a `var/examplename = (dildo_shape == "knotted") ? "They swallowed the knot" : "Their face is turning blue"`, then replace the involved phrase with `[examplename]`. and delete the subtype's override of the proc.

deathride58 commented on modular_citadel/code/modules/arousal/toys/dildos.dm:
```diff
@@ -131,3 +134,46 @@ obj/item/dildo/custom
+	user.Stun(150)
+	user.adjust_blurriness(8)
+	user.adjust_eye_damage(10)
+	addtimer(CALLBACK(src, .proc/manual_suicide, user), 15)
```
actually waitwaitwaitwait do_after() would be much more appropriate for this

Ragolution committed:
Code Improvements (again)

Knowing what I know now makes me wanna go revamp every instance of suicide code but geez.

Ragolution committed:
oh whoops

Ragolution committed:
Removed knot conditionals

whatever

Ghommie commented on modular_citadel/code/modules/arousal/toys/dildos.dm:
```diff
@@ -138,16 +138,15 @@ obj/item/dildo/custom
 		user.death(0)
 
 /obj/item/dildo/suicide_act(mob/living/user)
+//	is_knotted = ((src.dildo_shape == "knotted")?"They swallowed the knot":"Their face is turning blue")
```
Yea, Reason why you did got errors previously was because a var operation like this needs to be contained within the proc.
So, now that you know this, you can uncomment this code, add the var/ prefix, and readd it to the flavour text if you want.
Otherwise remove this comment.

Ragolution replied:
That was what I tried and it gave me issues, such as the triggering character not actually dying. It displays the message correctly but it seems to break everything else. I'm just going to try to committ this, for now, as it is. People can use their imagination if they want to choke on dog-dick, though I appreciate this all for the exercise in coding it was.
